contracts/base/ExecutionHelper.sol

```diff
@@ -60,7 +60,7 @@ contract ExecutionHelper is IExecutionHelperEventsAndErrors {
         assembly {
             result := mload(0x40)
             calldatacopy(result, callData.offset, callData.length)
-            success := iszero(call(gas(), target, value, result, callData.length, codesize(), 0x00))
+            success := call(gas(), target, value, result, callData.length, codesize(), 0x00)
```

Comment:
can you open a PR for this on reference-implementation repo please and describe in PR discussion

Reply:
done ✔️ https://github.com/erc7579/erc7579-implementation/issues/31